nix/editor.nix

```diff
@@ -0,0 +1,77 @@
+let
+  pkgs = import
+    (builtins.fetchTarball {
+      url = "https://github.com/NixOS/nixpkgs/archive/a78de4f116747585ebdb6f3d725268a7e068c554.tar.gz";
```

Comment:
The reason not to use the top-level nixpkgs is that every time we update that (we should hopefully be ~ once a week?) this might need to be rebuild for all users, and we probably want a slower cadence then that.

Reply:
Good point.

It seems like VSCodium uses it's own deno version. So I guess it's between using the nixpkgs from flake.nix and getting matching denos, or using a different one and getting less recompilation. (I don't feel strongly one way or the other.)

Reply:
There are some complications with that. If it's in the `flake` file, then we either make it a dependency of `garn` (downside: everyone has to download it when installing garn, even if they never use it), or we have to reference the entire directory from `garn` (basically, include the entire repo in `data-files`) so that the references of `flake.nix` work.

If we make it a separate file, then we have to use `nixpkgsInput` and hard code the sha. This is probably the best option, but if it's okay I'm going to leave it for later so I can get the release out.